src/components/views/directory/NetworkDropdown.tsx

```diff
@@ -41,7 +41,8 @@ import QuestionDialog from "../dialogs/QuestionDialog";
 import UIStore from "../../../stores/UIStore";
 import { compare } from "../../../utils/strings";
 
-export const ALL_ROOMS = Symbol("ALL_ROOMS");
+// XXX: We would ideally use a symbol here but we can't since we save this value to localStorage
```

Comment:
You can, just call `String(symbol)` on it

Reply:
oh but then rehydrating that symbol is a pain, nvm ignore this

Reply:
But I would probably need some silly logic for reading from LS, right? `localStorage.getItem(key)` returns a string, so I would need to convert it to a symbol then but I would need to do this only if the the return value was `"ALL_ROOMS"`